src/blog/2025-06-01-oxlint-stable.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Oxlint 1.0
+outline: deep
+authors:
+  - camc314
+---
+
+<AppBlogPostHeader />
+
+After eighteen months of dedicated development and hundreds of contributions from our community, Oxlint 1.0 is here! This milestone represents our commitment to building a production-ready linter that developers can rely on.
```

Comment:
Need to drill in the idea:

* we are finally stable and we mean it
* we have dedicated engineer working on the linter
* please try oxlint in your codebase and report issues, we will respond.